packages/components/addon/components/hds/gutters/index.js

```diff
@@ -0,0 +1,95 @@
+import Component from '@glimmer/component';
+import { htmlSafe } from '@ember/template';
+import { assert } from '@ember/debug';
+
+export const DIRECTIONS = ['horizontal', 'vertical'];
+export const SPACING_SIZES = ['xxs', 'xs', 'sm', 'md', 'lg', 'xl', 'xxl'];
+
+const regEx = /^((\d+)|(\d+\.\d+)|(\.\d+))(px|rem|em|%)$/;
```

Comment:
This is to check for valid css units consisting of a number and some of the most common CSS units. We may want to do something more sophisticated and/or support more types of units.